Kickstarter-iOS/Views/Controllers/PledgeShippingLocationViewController.swift

```diff
@@ -117,6 +121,13 @@ final class PledgeShippingLocationViewController: UIViewController {
         )
       }
 
+    self.viewModel.outputs.isLoading
```

Comment:
This output was still dangling but I'll add separate outputs for hiding and showing these.